geomstats/geometry/discrete_surfaces.py

```diff
@@ -931,7 +931,92 @@ def path_energy(self, path):
         """
         return 0.5 * gs.sum(self.path_energy_per_time(path))
 
-    def _ivp(self, initial_point, initial_tangent_vec, times):
+    def exp(self, tangent_vec, base_point):
+        """Compute the exponential map.
+
+        Parameters
+        ----------
+        tangent_vec : array-like, shape=[..., dim]
+            Tangent vector at base point.
+        base_point : array-like, shape=[..., dim]
+            Base point.
+
+        Returns
+        -------
+        exp : array-like, shape=[..., dim]
+            End point of the geodesic starting at base_point with
+            initial velocity tangent_vec and stopping at time 1.
+        """
+        return self.exp_solver.exp(self._space, tangent_vec, base_point)
```

Comment:
in terms of API, is `exp_solver.exp` a bit redundant?

why not `exp_solver()` directly? or `solver.exp()` ?